appveyor.yml

```diff
@@ -20,300 +19,40 @@ environment:
-     }
-     # Install ssh stuff.
-     copy ssh_id_rsa $env:userprofile\.ssh\id_rsa
-     if ($errorCode -ne 0) { $host.SetShouldExit($errorCode) }
-    }
- - type ssh_known_hosts >> %userprofile%\.ssh\known_hosts
- - cd ..
+ - ps: appveyor\scripts\setBuildVersionVars.ps1
+ - ps: appveyor\scripts\decryptFilesForSigning.ps1
  - git submodule update --init
 
 build_script:
- - ps: |
-     $sconsOutTargets = "launcher developerGuide  changes userGuide client"
-     if(!$env:APPVEYOR_PULL_REQUEST_NUMBER) {
-      $sconsOutTargets += " appx"
-     }
-     $sconsArgs = "version=$env:version"
-     if ($env:release) {
-      $sconsArgs += " release=1"
-     }
-     if ($env:versionType) {
-      $sconsArgs += " updateVersionType=$env:versionType"
-     }
-     $sconsArgs += ' publisher="NV Access"'
-     if(!$env:APPVEYOR_PULL_REQUEST_NUMBER) {
-      $sconsArgs += " certFile=appveyor\authenticode.pfx certTimestampServer=http://timestamp.digicert.com"
-     }
-     $sconsArgs += " version_build=$env:APPVEYOR_BUILD_NUMBER"
-     # We use cmd to run scons because PowerShell throws exceptions if warnings get dumped to stderr.
-     # It's possible to work around this, but the workarounds have annoying side effects.
-     Set-AppveyorBuildVariable "sconsOutTargets" $sconsOutTargets
-     Set-AppveyorBuildVariable "sconsArgs" $sconsArgs
- - 'echo scons args: %sconsArgs%'
+ - ps: appveyor\scripts\setSconsArgs.ps1
  - scons source %sconsArgs%
- - if ERRORLEVEL 1 exit %ERRORLEVEL%
- # We don't need launcher to run checkPot, so run the checkPot before launcher.
- - ps: |
-     cmd.exe /c "scons checkPot $sconsArgs"
-     if($LastExitCode -ne 0) {
-      Set-AppveyorBuildVariable "testFailExitCode" $LastExitCode
-      Add-AppveyorMessage "FAIL: Translation comments check. Translation comments missing or unexpectedly included. See build log for more information."
-     } else {
-      Add-AppveyorMessage "PASS: Translation comments check."
-     }
- # The pot gets built by tests, but we don't actually need it as a build artifact.
- - 'echo scons output targets: %sconsOutTargets%'
  - scons %sconsOutTargets% %sconsArgs%
- - if ERRORLEVEL 1 exit %ERRORLEVEL%
- # Build symbol store.
- - ps: |
-     foreach ($syms in
-      # We don't just include source\*.dll because that would include system dlls.
-      "source\liblouis.dll", "source\*.pdb",
-      "source\lib\*.dll", "source\lib\*.pdb",
-      # We include source\lib64\*.exe to cover nvdaHelperRemoteLoader.
-      "source\lib64\*.dll", "source\lib64\*.exe", "source\lib64\*.pdb",
-      "source\synthDrivers\*.dll", "source\synthDrivers\*.pdb"
-     ) {
-      & $env:symstore add /s symbols /compress -:NOREFS /t NVDA /f $syms
-     }
- - cd symbols
- - 7z a -tzip -r ..\output\symbols.zip *.dl_ *.ex_ *.pd_
- - if ERRORLEVEL 1 exit %ERRORLEVEL%
- - cd ..
+ - ps: appveyor\scripts\buildSymbolStore.ps1
+ - 7z a -tzip -r output\symbols.zip symbols\*.dl_ symbols\*.ex_ symbols\*.pd_

```

Comment:
@seanbudd As this command now manually mentions the symbols directory in each file to be included, rather than changing directory to symbols first, the zip file now contains a subdirectory called symbols with everything inside. 
This is not what our server expects, so when the zip file is extracted, we end up with a `symbols` directory within our symbol server.
We should change this line back to
```
cd symbols
7z a -tzip -r ..\output\symbols.zip *.dl_ *.ex_ *.pd_
```
Though I don't know the full reasoning as to why changing directories was avoided here.

Reply:
This was just to simplify code. I will revert this.